src/components/Book.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prop-types */
+import { useDispatch } from 'react-redux';
```

Comment:
- [ ] Kindly don't prevent linters from raising those errors. Kindly fix them manually instead of disabling the linters because linters shows you those errors which can break your entire application 😨